analyzer/src/main/kotlin/managers/Yarn.kt

```diff
@@ -69,6 +70,8 @@ class Yarn(
 
     override fun getRemotePackageDetails(workingDir: File, packageName: String): JsonNode {
         val process = run(workingDir, "info", "--json", packageName)
-        return jsonMapper.readTree(process.stdout)["data"]
+        return jsonMapper.readTree(process.stdout)["data"] ?: jsonMapper.readTree(process.stderr)["data"].also {
+            log.warn { "Error running `${process.commandLine}` in directory $workingDir: ${jsonMapper.readTree(process.stderr)["data"]}" }
```

Comment:
`also` makes its context available as `it` inside the lamda, so `${jsonMapper.readTree(process.stderr)["data"]}` should simply become `$it` here.

Reply:
Or you do `...also { yarnErr ->`, which is what @mnonnenmacher probably forgot in his example.

Reply:
done